OG_Format.adoc

```diff
@@ -347,8 +347,8 @@ OG1.0 requirements cover the GPS variables delivered by the glider when at the s
 * data type: double
 * dimension: N_MEASUREMENTS |
 
-* long_name = “longitude of each GPS location”;
-* standard_name = “longitude”;
+* long_name = “Longitude east by unspecified GPS system”;
+* standard_name = “GPS fixed longitude”;
```

Comment:
I don't see `GPS fixed longitude` in the standard name table. The `standard_name` requires a valid entry. It is often used `longitude` (yes, identical to above since they are both longitudes, but different variables). One alternative is to register a new entry in the Standard Names table, it violates the CF rules as proposed.

Reply:
shall we leave this blank for now until a new entry is created

Reply:
I don't think we need a new entry do we ? I think we just have to align to what exist.
LAT_GPS and LON_GPS are well described in NVS. 
The 'concept name' in the nvs is "GPS fixed longitude" and "GPS fixed "latitude". See https://vocab.nerc.ac.uk/collection/OG1/current/LON_GPS/. 
Maybe the "concept name" in NVS should not be our standard name. Then what should be our standard_name when the vocab exist ? 

Let's use correctly what is available at NVS.

I push this discussion #271, as I think, the issue comes from a bit of unclarity in the use of the content of the NVS.